algotrading/decisionmodel/random_decision.py

```diff
@@ -1,14 +1,18 @@
 import random
 
+
 def random_decision_long(df, feature_columns):
```

Comment:
are these feature_columns vars used for anything?

Reply:
yeah so they are intended to be used by the decision models.  

the idea is that feature_columns tells the decision model which columns that it should use to make decisions.  could be a better convention and/or name for that.

the random decision model, which is just sort of a dummy baseline model, doesn't use it because it basically just flips a coin to decide what to do.